clients/java/src/main/java/io/camunda/client/impl/command/UnassignRoleFromUserCommandImpl.java

```diff
@@ -22,6 +22,7 @@
 import io.camunda.client.api.response.UnassignUserFromRoleResponse;
 import io.camunda.client.impl.http.HttpCamundaFuture;
 import io.camunda.client.impl.http.HttpClient;
+import io.camunda.client.impl.response.UnassignUserFromRoleResponseImpl;
```

Comment:
💭 this class is called `UnassignRoleFromUser` but the req/res objects are called `UnassignUserFromRole` 🤔

Reply:
I think they had a discussion and wanted to make this consistent 🤷